static/index.html

```diff
@@ -1,23 +1,53 @@
 <!DOCTYPE html>
 <html lang = "en">
-<head>
- <meta charset="utf-8">
- <meta http-equiv="X-UA-Compatible" content="IE=edge">
- <meta name="viewport" content="width=device-width, initial-scale=1">
- <title>Drupalcamp Cebu 2018</title>
-
- <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
- <link rel="stylesheet" href="css/style.css">
-<link href="https://fonts.googleapis.com/css?family=Merriweather:400,700|Open+Sans" rel="stylesheet">
- <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
-
- <meta property="og:url" content="" />
- <meta property="og:title" content="Drupalcamp Cebu 2018" />
- <meta property="og:description" content="Drupalcamp Cebu is a one day conference about Web development with sessions about Drupal, DevOPS, UI/UX and Business." />
- <meta property="og:image" content="" />
-</head>
+  <head>
+   <meta charset="utf-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1">
+   <title>Drupalcamp Cebu 2018</title>
+   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+   <link rel="stylesheet" href="css/style.css">
+   <script src="script/mobile-menu.js"></script>
+   <link href="https://fonts.googleapis.com/css?family=Merriweather:400,700|Open+Sans" rel="stylesheet">
+   <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+   <meta property="og:url" content="" />
+   <meta property="og:title" content="Drupalcamp Cebu 2018" />
+   <meta property="og:description" content="Drupalcamp Cebu is a one day conference about Web development with sessions about Drupal, DevOPS, UI/UX and Business." />
+   <meta property="og:image" content="" />
```

Comment:
Please complete the og:image. The image should be representing the camp, you can use the photo group from last year. For the size, please refer to FB recommendation: https://developers.facebook.com/docs/sharing/best-practices/#images

Reply:
Done